Makefile.core.mk

```diff
@@ -77,7 +77,7 @@ baseurl := "$(URL)"
 endif
 
 # Which branch of the Istio source code do we fetch stuff from
-export SOURCE_BRANCH_NAME ?= master
+export SOURCE_BRANCH_NAME ?= release-1.25
```

Comment:
Shouldn't we leave `master` refs as-is? Typically we do, right?

Or is this a temporary chore we need to do, and then afterwards flip it back from `release-1.25` to `master`?

Reply:
Yes, I think we move this back later

Reply:
Oh yep: https://github.com/istio/istio.io/pull/15848/files